packages/starlight/__e2e__/collection-config.test.ts

```diff
@@ -1,6 +1,6 @@
 import { expect, testFactory } from './test-utils';
 
-const test = await testFactory('./fixtures/custom-src-dir/');
+const test = await testFactory('./fixtures/custom src-dir/');
```

Comment:
Small detail: should we add a comment mentioning why there is a space in the fixture name and that we use it to test this specific case, considering it may be the only fixture with a space in the name so we don't remove it by mistake?

Reply:
Yes, good idea, thank you!

```suggestion
// This fixture contains a space in the directory so that we have a smoke test for building
// Starlight projects with pathnames like this, which are a common source of bugs.
const test = await testFactory('./fixtures/custom src-dir/');
```